arm9/src/scrcmd_3.c

```diff
@@ -0,0 +1,467 @@
+#include "global.h"
+
+#include "heap.h"
+#include "map_header.h"
+#include "party.h"
+#include "pokedex.h"
+#include "pokemon_storage_system.h"
+#include "save_vars_flags.h"
+#include "scrcmd.h"
+
+extern void *FieldSysGetAttrAddr(struct FieldSystem *, u8 idx);
+extern SaveFashionData *Save_FashionData_Get(SaveData *save);
+
+extern void sub_02082140(void *, u8);
```

Comment:
these all should have parameter names, (param0, param1 etc is preferred if you don't know what they are, sometimes it's obvious, but sometimes it's not)